update_reminder.sh

```diff
@@ -1,10 +1,73 @@
 #!/bin/bash
-MARKT_LAST_CHANGE=$(wget -q -O- https://wo-ist-markt.de/cities/berlin.json | sed -n 's/.*aktualisiert am \(.*\)",/\1/p')
-BERLIN_LAST_CHANGE=$(wget -q -O- https://daten.berlin.de/datensaetze/berliner-wochen-und-tr%C3%B6delm%C3%A4rkte | sed -n '/.*Aktualisiert.*/,$p' | sed -e '/.*end.*/,$d' | sed -n 's/.*<span .*>\(.*\)<\/span.*/\1/p')
-
-if [[ "$MARKT_LAST_CHANGE" != "$BERLIN_LAST_CHANGE" ]]; then
-  echo "Dates don't match!"
-  echo "daten.berlin.de: $BERLIN_LAST_CHANGE"
-  echo "wo-ist-markt.de: $MARKT_LAST_CHANGE"
-  exit 1;
-fi
+
+set -e #exit on error
+
+#Collect errors and fail in the end
+GOT_ERROR=false
+
+#get HTTP code from site and exit 1 iff it is not 200
```

Comment:
Typo: `iff`

Reply:
It was just a shorthand for "if and only if". I spell it out now